### ISSUE TITLE
like/dislike songs

### ISSUE THREAD

**Issue by dionorgua** [OzymandiasTheGreat/mopidy-ytmusic#23]
Hi,

It would be cool to have a way to like/dislike songs. Maybe as handler of 'add to liked playlist' event, or just by some special 'mopidy ytmusic like/dislike' command.

**Comment by OzymandiasTheGreat** [comment]
I thought about this but decided against it. I mean a command would make sense if you're sitting at the terminal but that's it.
Mopidy doesn't really provide a sensible way to implement this. If we went the special playlist way, just think how many steps it would take to like a song with most clients.
This wouldn't be difficult to implement I think but the usabillity would be sub-par.

**Comment by dionorgua** [comment]
I agree that it's not easily implemented right now. At the same time 'special' playlist or some sort of custom like command opens at least some way to do this. With some scripting it at least solves this somehow. 

For example for me it's one of key feature of such 'alternative' client because it improves quality of suggestions.

I think this one is related:
https://github.com/mopidy/mopidy/issues/327
